app/services/symphony_client.rb

```diff
@@ -28,6 +28,46 @@ def patron_info(user, item_details: {})
     JSON.parse(response.body)
   end
 
+  def change_pickup_library(hold_key:, pickup_library:, session_token:)
+    response = authenticated_request('/circulation/holdRecord/changePickupLibrary',
+                                     headers: { 'x-sirs-sessionToken': session_token },
+                                     method: :post, json: {
+                                       holdRecord: {
+                                         resource: '/circulation/holdRecord',
+                                         key: hold_key
+                                       },
+                                       pickupLibrary: {
+                                         resource: '/policy/library',
+                                         key: pickup_library
+                                       }
+                                     })
+
+    response
+  end
+
+  # not working!
+  # {
+  #   "messageList":[
+  #     {
+  #       "code": "permissionDeniedEditedFieldNotEditable",
+  #       "message": "PUT Permission denied: Field fillByDate not editable."
+  #     }
+  #   ]
+  # }
```

Comment:
these can be deleted right now that it is working?

Reply:
oops sorry!